src/components/board/Board.jsx

```diff
@@ -3,7 +3,7 @@ import { ICONS } from "@/data/icons";
 
 const Board = ({ name, position, grade, major, links, pronouns, picture }) => {
   return (
-    <div className="w-full bg-gray-200 my-4 shadow-[12px_-12px_0px_0px] shadow-ewb-green p-4">
+    <div className="w-full lg:col-span-4 md:col-span-3 bg-gray-200 shadow-[12px_-12px_0px_0px] shadow-ewb-green p-4">
```

Comment:
you need to adjust the grid-col-[number] in the parent component. The way you change it will make it only one col when the screen is big, which is the opposite of what it should behave.